docs/05-low-code-spark/05-gems/02-transform/01-reformat.md

```diff
@@ -3,8 +3,71 @@ sidebar_position: 1
 title: Reformat
 ---
 
-:::caution 🚧 Work in Progress 🚧
+Edit one or more column names or values, by using expressions and functions. It's useful when we need to extract only the required columns or make changes column-wise.
 
-TODO
 
+### Parameters
+| Parameter        | Meaning                                       | Required                                     |
+|:-----------------|:----------------------------------------------|:---------------------------------------------|
+| Dataframe        | Input dataframe on which changes are required | True                                         |
+| Target column    | Output column name                            | False                                        |
+| Expression       | Expression to compute target column           | False (Required if target column is present) |
+
+:::info 
+To quickly select all the columns, click on the ADD ALL on the left side bar. 
+:::
+:::info
+If no columns are selected, then all columns are passed through to the output
 :::
+
+### Example
+
+![](./img/reformat_eg_1.png)
```

Comment:
Always include Alt text (inside the square brackets) for accessibility reasons

Reply:
fair